mesh/meshdb.go

```diff
@@ -658,3 +656,22 @@ func LayerIDs(db database.Database, namespace string, lid types.LayerID, f func(
 	}
 	return database.ErrNotFound
 }
+
+type beaconDB struct {
```

Comment:
and this too

Reply:
already removed, but after I requested your review